packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -353,6 +353,14 @@ class ObjectsInRedisClient {
                                         );
                                         this.settings.disconnected();
                                     }
+                                } else if (channel === `${this.metaNamespace}objects.features.useSets`) {
+                                    const newUseSets = !!parseInt(message);
+                                    if (newUseSets !== this.useSets) {
+                                        this.log.info(
+                                            `${this.namespace} Sets ${newUseSets ? 'activated' : 'deactivated'}`
+                                        );
+                                        this.useSets = newUseSets;
+                                    }
```

Comment:
im activate fall vllt doch wenigstens nen restart machen oder klar loggen "restarte mich" weil er bekommt ja keine sets neu erstellt ?!

Reply:
oder vllt aktivieren udn dann restart mit 10s verzögerung das idealerweise die sets schon da sind ...

Reply:
ja restart oder lua neu einlesen muss in beiden fällen sein

Reply:
durch die falschen luas würden keine object views mehr laufen, also best case wäre echt einfach disconnect